Makefile

```diff
@@ -0,0 +1,45 @@
+SYMFONY_ENV = dev
+PHP_RUN = php7.4
+COMPOSER_PATH = /usr/local/bin/composer2
+ifeq ("$(wildcard $(COMPOSER_PATH))","")
+	COMPOSER_PATH = /usr/local/bin/composer
+endif
+COMPOSER_RUN = $(PHP_RUN) $(COMPOSER_PATH)/composer.phar
```

Comment:
@RandyCupic path pattern to composer is not common for most of our environments, right?
Usually /usr/local/bin/composer(2) IS the executable, and not a folder with phar file inside

It can be adjusted according to the project or environment requirements, but definitely not common.

Reply:
@hknezevic It appears that I had a bad setup on my local instance so it was not working for me (`/usr/local/bin/composer` was a folder, not an executable). I've fixed this in https://github.com/netgen/media-site/pull/81/commits/dd07dab73d0be931f1bb7548e56f36ddf75d3404.